mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -211,18 +215,23 @@ CLASS net/minecraft/unmapped/C_ayfeobid net/minecraft/client/MinecraftClient
 		ARG 2 monitor
 	METHOD m_bteytekh render (Z)V
 		ARG 1 tick
+	METHOD m_buwtrqcs setLastInputType (Lnet/minecraft/unmapped/C_qdenxghm;)V
+		ARG 1 lastInputType
 	METHOD m_cbjyfsge refreshSearch (Lnet/minecraft/unmapped/C_zqzigdge$C_gvofkqkn;Ljava/util/List;)V
 		ARG 1 key
 		ARG 2 searchables
 	METHOD m_ccoxkwrp (Ljava/util/function/Consumer;Ljava/nio/file/Path;)V
 		ARG 1 path
 	METHOD m_cfvjiijv cancelRecorder ()V
 	METHOD m_crapepyd getEntityModelLoader ()Lnet/minecraft/unmapped/C_qncyfzro;
+	METHOD m_cwwicafr isSinglePlayer ()Z
```

Comment:
not a comment on this pr, but I'm seeing `SinglePlayer` and `Singleplayer` interspersed throughout qm. would be good to clean that up

Reply:
I personally think "Singleplayer" is the superior capitalization, although my phone thinks otherwise lol.

Reply:
^ I would agree there

Reply:
```suggestion
	METHOD m_cwwicafr isSingleplayer ()Z
```